src/lib/types/passage.ts

```diff
@@ -1,38 +1,25 @@
-import type { MediaType } from './resource';
+import type { ResourceContentInfo } from './resource';
 
 export enum PassagePageMenuType {
     mainMenu = 'mainMenu',
     guideMenu = 'guideMenu',
 }
 
-export interface BasePassage {
+export interface BibleSection {
```

Comment:
So much of what we had was tied to passages, which is a specific concept on the backend. I renamed a lot of it to make it more generic.